pkg/generated/openapi/cmd/models-schema/main.go

```diff
@@ -42,6 +43,18 @@ func output() error {
 	defs := openapi.GetOpenAPIDefinitions(refFunc)
 	schemaDefs := make(map[string]spec.Schema, len(defs))
 	for k, v := range defs {
+		// Replace top-level schema with v2 if a v2 schema is embedded
```

Comment:
I don't understand why this is part of the PR?

Reply:
tl;dr to ensure this program outputs valid OpenAPI v2. Before this PR, this program outputs v3 which just happened to be valid v2. After this PR this coincidence is no longer valid.